src/harness/certs/generate_fake_certs.sh

```diff
@@ -151,6 +152,65 @@ openssl ca -cert unknown_ca.cert -keyfile private/unknown_ca.key -in unknown_dev
     -batch -notext -create_serial -utf8 -days 1185 -md sha384
 
 
+# Generate certificates for test case WINNF.FT.S.SCS.6 - Unrecognized root of trust certificate presented during registration
+echo "\n\nGenerate 'unrecognized_device' certificate/key"
+openssl req -new -x509 -newkey rsa:4096 -sha384 -nodes -days 7300 \
+    -extensions root_ca -config ../../../cert/openssl.cnf \
+    -out unrecognized_root_ca.cert -keyout private/unrecognized_root_ca.key \
+    -subj "/C=US/ST=CA/L=Somewhere/O=Generic Certification Organization/OU=www.example.org/CN=Generic RSA Root CA"
+
+openssl req -new -newkey rsa:2048 -nodes \
+    -reqexts cbsd_req -config ../../../cert/openssl.cnf \
+    -out unrecognized_device.csr -keyout unrecognized_device.key \
+    -subj "/C=US/ST=CA/L=Somewhere/O=Generic Certification Organization/OU=www.example.org/CN=Unrecognized CBSD"
+openssl ca -cert unrecognized_root_ca.cert -keyfile private/unrecognized_root_ca.key -in unrecognized_device.csr \
+    -out unrecognized_device.cert -outdir ./root \
+    -policy policy_anything -extensions cbsd_req_sign -config ../../../cert/openssl.cnf \
+    -batch -notext -create_serial -utf8 -days 1185 -md sha384
+
+#Certificate for test case WINNF.FT.S.SCS.8 - Self-signed certificate presented during registration
+#Using the same CSR that was created for normal operation
+echo "\n\nGenerate 'self_signed_client' certificate/key"
+openssl x509 -signkey client.key -in client.csr \
+    -out self_signed_client.cert \
+    -req -days 1185
+
```

Comment:
generate the 'corrupted_client' here, as you did for other certificate?
could be like:

```
# Certificates for test case WINN.FT.S.SCS.7 - corrupted certificate, based on client.cert
cp client.key corrupted_client.key
cp client.cert corrupted_client.cert
  # cert file have first header line with 28 char: we want to change the 10th cert character
pos=38
hex_byte=$(xxd -seek $((10#$pos)) -l 1 -ps corrupted_client.cert -)
  # increment this byte
corrupted_dec_byte=$(($((16#$hex_byte)) +1))
  # write it back
printf "%x: %02x" $pos $corrupted_dec_byte | xxd -r - corrupted_client.cert

```